claude/README.md

```diff
@@ -0,0 +1,381 @@
+# Claude API Module for sttp-openai
```

Comment:
do we need two readmes? I think it would be better to have one, probably with a ToC

Reply:
_merged_